src/main/java/org/zalando/nakadi/service/kpi/NakadiPublishingController.java

```diff
@@ -0,0 +1,29 @@
+package org.zalando.nakadi.service.kpi;
+
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.zalando.nakadi.service.kpi.publisher.NakadiKpiPublisher;
+
+import java.util.function.Supplier;
+
+@Service
+public class NakadiPublishingController {
```

Comment:
Can we have a different name than a "controller"? In a context of Spring the "controller" usually means the handler of some URL requests.

Reply:
@v-stepanov name it :)

Reply:
To say the truth, I don't understand the purpose of this class. For me it seems that NakadiKpiPublisher could be directly called from LoggingFilter. 
That's why I can't find a name for it :)

Reply:
I thought about it as well, but decided to keep in order to collect related calls to publish in one place and do not spread them all over the code.